libs/hmpb/src/ballot_components.tsx

```diff
@@ -233,19 +233,19 @@ export function Bubble({
   );
 }
 
-export function BubbleWrapper({
+export function AlignedBubble({
   optionInfo,
-  style = {},
 }: {
   optionInfo: OptionInfo;
-  style: React.CSSProperties;
 }): React.ReactElement {
   return (
     <div
       style={{
         display: 'flex',
         alignItems: 'center',
-        ...style,
+        // Match line-height of text to align bubble to center of first line of
+        // option label
+        height: '1.2rem',
```

Comment:
Seems brittle - would it be easy to have this value be shared between here and wherever the line height is set?
(I'm guessing the latter isn't likely to change, so feel free to ignore if it's too much of a headache)

Reply:
Good idea, updated here: a21181836